systemtests/src/test/java/io/enmasse/systemtest/bases/mqtt/MqttPublishTestBase.java

```diff
@@ -122,7 +122,8 @@ private void publish(List<MqttMessage> messages, int subscriberQos) throws Excep
         setAddresses(dest);
 
         MqttConnectOptions options = new MqttConnectOptions();
-        options.setConnectionTimeout(options.getConnectionTimeout() * 2);
+        options.setConnectionTimeout(options.getConnectionTimeout() * 2);  //Default is 30 seconds, increase it to 1 min.
+        options.setKeepAliveInterval(options.getKeepAliveInterval() * 2);  //Default is 60 seconds, increase it to 2 min.
```

Comment:
I think keepAlive timeout is the frequency of the PINGREQ/PINGRESP packets.  Not sure if tuning this helps this problem

Reply:
I wasn't sure either.  Removed it for now.